ts/src/db.atlas.connect.ts

```diff
@@ -1,19 +1,19 @@
 import mongoose from 'mongoose';
 import dotenv from 'dotenv';
 import { server, watchEslint } from './lib/script';
-
+import chalk from "chalk";
```

Comment:
@Dkingofcode use single quotes in this chalk import. Not double quotes.

Reply:
Also, Change your pull request title according to how it is instructed in this page of the community docs: https://code-collabo.gitbook.io/community-doc/collabo-guidelines/contributing @Dkingofcode